docs/governance-strategy.md

```diff
@@ -0,0 +1,137 @@
+
+:::
+
+Below is a bootstrap plan for encoding the Executable Books team policy and decision-making process. Its goal is to be as simple as possible and not deviate too much from our current practices (to maximize the chances of being followed). It also aims to balance "formal process" against "efficient asynchronous decision-making" (to avoid being so slow that it hinders the organization).
+
+Over time, as we learn what works and what doesn't, we may make amendments to this process to make it more complex.
+
+:::{admonition} Acknowledgements
+This builds off of the work that Chris Sewell did in creating the current [MyST Enhancement Proposals repository](https://myst-eps.readthedocs.io/en/latest/mep-0001.html). It takes much of the process there, and tries to simplify and slim it down. It also separates out two different processes - a more complex one for MyST, and a simpler one for organizational policy. We can revisit this in the future if we believe that we _need_ more complexity in one or the other. Many thanks to Chris S (and others) for feedback and inspiration for these ideas.
+:::
+
+## Groups with decision-making power in the Executable Books organization
+
+### Steering Council
+
+The group of people with decision-making authority over the entire organization. Their primary duty is to set organizational policy and strategy, to steward all technical and IP assets of the organization, to make decisions when we are at an impasse, and to delegate decision-making power to others in the organization. Members of the steering council all have "owner" status of the [executablebooks](https://github.com/executablebooks) GitHub organizations and repositories, and are the only individuals with "owner" status.
+
+The Steering Council is currently comprised of the three Principle Investigators of the original grant funding the project: [Chris Holdgraf](https://github.com/choldgraf), [John Stachurski](https://github.com/jstac), and [Greg Caporaso](https://github.com/gregcaporaso). We expect the Steering Council to expand in the future, and this initial Steering Council will define policies that set the expectations for steering council membership as well as the process for modifying or expanding the steering council membership in the future.
+
+### Core team members
+
+Individuals who are particularly interested in the EBP community and have demonstrated a willingness to participate in our community and further its mission. They guide discussion, grow the community, contribute code, and generally help the project improve.
+
+Currently the core team is [defined here](https://executablebooks.org/en/latest/team.html).
+
```

Comment:
Maybe a good place to add a reference to the code of conduct. For example:

```markdown
## Code of Conduct
Steering Council and Core Team Members agree to abide by the [EBP Code of Conduct](https://github.com/executablebooks/.github/blob/master/CODE_OF_CONDUCT.md). It is our expectation that _all contributors_ to the Executable Books Project (including but not limited to individuals contributing code, providing technical support, teaching workshops, and engaging in discussions about changes to EBP policy or the MyST Specification) adhere to the Code of Conduct.  
```

There's probably a better link to the CoC - I'm not sure what it is.

Reply:
Agree that the Code of Conduct needs to be in a prominent place in any spot that describes the policies of the team. I think that we should have a link to the CoC in both the `executablebooks.org` and `compass.executablebooks.org` spot, but agreed that the "team compass" is where the Code of Conduct should be defined (or where it delegates the COC to another place)